src/serverlist.cpp

```diff
@@ -41,7 +42,18 @@ CServerListManager::CServerListManager ( const quint16  iNPortNum,
     SetCentralServerAddress ( sNCentServAddr );
 
     // set the server internal address, including internal port number
-    SlaveCurLocalHostAddress = CHostAddress( NetworkUtil::GetLocalAddress().InetAddr, iNPortNum );
+    QHostAddress qhaServerPublicIP;
```

Comment:
Is `qhaServerPublicIP` the proper naming scheme?

Reply:
There generally isn't consistent Microsoft-style class name initial prefixing of variable names.  So these aren't inherently "right" (and I bristle against them) but there are enough type-prefixed names in the code to make them not inherently "wrong", either.